server/models/Schedule.js

```diff
@@ -0,0 +1,87 @@
+
+const mongoose = require("mongoose");
+
+const scheduleSchema = new mongoose.Schema({
+  name: {
+    type: String,
+    required: true,
+  },
+  active: {
+    type: Boolean,
+    required: true,
+  },
+  periods: {
+    monday: {
+      startTime: {
+        type: Date,
```

Comment:
Thinking about this now - it wouldn't necessarily be a date type being stored here. We're asking for 24 hour time rather than a date. We'll have logic in the back-end to make sure this is proper and works as intended but for the store we can just store it as a string